pmm/pmm2-upgrade-tests.groovy

```diff
@@ -203,35 +199,35 @@ pipeline {
             '''
             destroyStaging(VM_NAME)
             sh '''
-                sudo chmod 777 -R pmm-app/tests/output
-                ./pmm-app/node_modules/.bin/mochawesome-merge pmm-app/tests/output/parallel_chunk*/*.json > pmm-app/tests/output/combine_results.json
-                ./pmm-app/node_modules/.bin/marge pmm-app/tests/output/combine_results.json --reportDir pmm-app/tests/output/ --inline --cdn --charts
+                sudo chmod 777 -R tests/output
```

Comment:
Make a false pass for this part, incase of any failure during setup this will not error out and crash the pipeline execution, `|| true` just like we do on nightly and UI tests.

Reply:
done